python/src/pycryptosat.cpp.in

```diff
@@ -997,6 +997,38 @@ static PyObject* msolve_selected(Solver *self, PyObject *args, PyObject *kwds)
     return solutions;
 }
 
+PyDoc_STRVAR(get_conflict_doc,
+"get_conflict()\n\
+Returns the conflicts in the assumptions when the last call to solve(...)\n\
+was unsatisfiable.\n\
+\n\
+:return: Which assumptions (as passed to solve(...)) are incorrect\n\
+:rtype: <dict <int,boolean>>"
+);
+
+static PyObject* get_conflict(Solver *self)
+{
+    PyObject *result = PyDict_New();
+    const std::vector<Lit> conflicts = self->cmsat->get_conflict();
+
+    for (unsigned long i = 0; i < conflicts.size(); i++) {
+        Lit lit = conflicts[i];
+        #ifdef IS_PY3K
+        PyObject *key = PyLong_FromLong(lit.var()+1);
```

Comment:
This feels wrong. But, whenever I added an assumption in the Python API, `lit.var()` always comes back as one less than the lit I added.

Reply:
Of course it does :) Please read the README of the solver. Internally variables are numbered from 0. Please read the README.

Reply:
Mmmm ok. I see now, sorry. :)

Reply:
Haha no worries! I hope it clears it up :)